# try:
#     print(a)
# except:
#     print("error")
# else:
#     print("noerror")
# finally:
#     print("always")

a = 10
try:
    print("a"+10)
    print(a)
except Exception as lava:
    print("error",lava)